docs/data/sql_functions.yml

```diff
@@ -643,6 +643,9 @@ collection:
   - sql: ARRAY_REVERSE(haystack)
     table: haystack.arrayReverse()
     description: Returns an array in reverse order. If the array itself is null, the function will return null.
+  - sql: ARRAY_UNION(array1, array2)
+    table: haystack.arrayUnion(array)
```

Comment:
nit: `haystack` does not make sense in this context imo. It's a nice joke but imo works only for `ARRAY_CONTAINS`